spec/models/reporting_pipeline/patient_states_per_month_spec.rb

```diff
@@ -19,7 +19,7 @@
         patient_registered_13m_ago = Timecop.freeze(13.months.ago) { create(:patient) }
         Timecop.freeze(13.months.ago) { create(:blood_pressure, patient: patient_registered_13m_ago) }
 
-        described_class.refresh
+        RefreshMaterializedViews.new.refresh_v2
```

Comment:
Hopefully this isn't too much slower in specs than just refreshing the described class.